### PR TITLE
fix(carousel): update button sizes

### PR DESCRIPTION
**Pull request by kennylam**
### Related Ticket(s)

#6170 

### Description

Updates spacing of the carousel buttons.

### Changelog

**Changed**

- remove button padding to avoid scaling of inner icon

### CODE REVIEW

**Comment by jeffchew** [comment]
@kennylam fyi, going to retarget this to the expressive theme feature branch.

**Comment by kennylam** [comment]
> @kennylam fyi, going to retarget this to the expressive theme feature branch.

Doh! Forgot about that, thanks @jeffchew.

**Review by jeffchew — DISMISSED** [review]
Thanks @kennylam !

**Review by jeffchew — DISMISSED** [review]
LGTM!

File: packages/styles/scss/components/carousel/_carousel.scss [hunk 1]
```diff
@@ -89,9 +89,13 @@
   }
 
   .#{$prefix}--btn.#{$prefix}--carousel__navigation__btn {
-    padding: calc((#{$container-02} - #{$icon-size-01}) / 2);
+    padding: 0;
     min-height: $container-02;
     width: $container-02;
```
Comment (oliviaflory) [inline packages/styles/scss/components/carousel/_carousel.scss:94]: @kennylam is there a container-03 that would equal 40px? The buttons are rendering at 32 x 32px now

Comment (kennylam) [reply packages/styles/scss/components/carousel/_carousel.scss]: @oliviaflory Good catch, I forgot to bump up the size after removing the padding.

**Review by oliviaflory — APPROVED** [review]
Looks good @kennylam, thanks!

**Review by jeffchew — APPROVED** [review]

**Pull request merged**